CONTRIBUTING.md

```diff
@@ -0,0 +1,128 @@
+* ``SampleID``: Unique sample identifier.
+* ``BarcodeSequence``: Unique barcode (i.e., index) sequence for each sample.
+* ``LinkerPrimerSequence``: Forward primer sequence (mock microbiomes only, include ``NA`` if this is not applicable).
+* ``ReversePrimer``: reverse primer sequence (mock microbiomes only, include ``NA`` if this is not applicable).
+* ``PrimerName``: Forward and reverse primer names. We recommend using the format ``Xf-Yr``, where ``X`` is the forward primer name and ``Y`` is the reverse primer name. For example, 515f-806r. (Mock microbiomes only, include ``NA`` if this is not applicable.)
+* ``Description``: Unique description for each sample, usually around 1-5 words.
+
+### ``source/taxonomy.tsv``
+This file lists the taxonomic and (when possible) strain affiliation of each strain added to the mock community, as well as its relative abundance. This file does not need to adhere to a particular taxonomic reference database, but please include as much information as possible (e.g., if this strain is available through a public repository, please list the repository strain ID). This information is usually provided by the developer(s) of the mock community.
+
+This file must be in "classic BIOM" (tab-separated text) format.
+
+### Expected taxonomy (``database-name/database-version/taxonomy.tsv``)
+Contains the known composition of the mock community (e.g., taxonomies or KEGG pathways), annotated according to a specific reference database. Compilation of expected composition data is not a trivial task, and requires careful review of database annotations to ensure that accurate annotations are applied to source data. See *Compiling expected taxonomy files* below for discussion of this topic.
+
+This file must be in "classic BIOM" (tab-separated text) format.
+
+### Database identifiers (``database-name/database-version/database-identifiers.tsv``; optional)
+Contributors may provide database identifiers associated with each member of the mock community (useful, for example, for identifying associated sequences in the reference database). Each taxonomic name listed in the ``taxonomy.tsv`` in the same directory must be included, and can have zero or more database identifiers associated with it. The taxonomic name and all database identifiers should be separated by tabs.
+
+## Classic BIOM-formatted tables
+
+Several files are described as being in classic biom-format tables. In these files, the first line must begin with the text ``#Taxonomy``, followed by a tab-separated list of one or more sample identifiers. All sample identifiers provided here must be present in ``sample-metadata.tsv``. Each subsequent line should begin with the taxonomic name, followed by a tab-separated list of the relative abundances in each sample. The relative abundances must sum to 1.000 (to three decimal places) for each sample.
```

Comment:
*Several files are described as being in classic biom-format tables.*

->

*Several files are described as being classic biom-format tables.*